cmd/minikube/cmd/config/enable.go

```diff
@@ -41,6 +41,15 @@ var addonsEnableCmd = &cobra.Command{
 		if err != nil {
 			exit.WithError("enable failed", err)
 		}
+		if addon == "dashboard" {
+			out.T(out.Waiting, "enabling metrics-server addon along with dashboard")
+			err = addons.SetAndSave(ClusterFlagValue(), "metrics-server", "true")
+			if err != nil {
+				exit.WithError("enable failed", err)
+			}
+			out.T(out.AddonEnable, "The 'metrics-server' addon is enabled")
```

Comment:
Maybe make it more obvious.
" Some dashboard features need metrics add-on. The metric add-on was enalbled automatically..."

Reply:
What about "require" instead of "need"?  Also, should I refactor the two "if" statements into a "switch" one?

Reply:
this is how it looks now:
```
./out/minikube addons enable dashboard
⌛  Some dashboard features require the metrics add-on
🌟  The metrics add-on was enabled automatically
🌟  The 'dashboard' addon is enabled
```